services/121-service/src/transaction-job-processors/transaction-job-processors.service.ts

```diff
@@ -253,10 +252,22 @@ export class TransactionJobProcessorsService {
       }
     }
 
-    // 3. Start the transfer, save error transaction on failure and return early
-    let safaricomDoTransferResult: DoTransferReturnType;
+    // 3. Store 'waiting' transaction before starting transfer, because of callback.
```

Comment:
Comment is confusing, since the function called does more.

I suspect it is intended to also update latest transaction and registration, so update comment accordingly.